scripts/run-clang-tidy-on-compile-commands.py

```diff
@@ -43,6 +40,9 @@
 import tempfile
 import threading
 import traceback
+
+import click
```

Comment:
why was this re-ordering/separatiom done? Is it python built-in vs pip-installed?

Reply:
Yes, is import auto-formatting done by vscode (which, I think, is using [isort](https://github.com/PyCQA/isort) as a backend).

I think that some improvement could be done in regard to python files style/formatting and checking. For start maybe we should add flake8/pyling to CI. It will catch e.g. not used imports. When editing python files I can see some errors from time to time.

I can try to add some checks for python files to restyled setup.